smart-contracts/testdata/contracts/v1/upgrading-inspect-module0.wat

```diff
@@ -0,0 +1,194 @@
+(module
+    ;; Imports
```

Comment:
Could you please have a comment here outlining what this module is for. There is something like this in the PR description, but that will not be readily available after it's merged.